Mathlib/CategoryTheory/CatCommSq.lean

```diff
@@ -39,6 +39,16 @@ variable (T : C₁ ⥤ C₂) (L : C₁ ⥤ C₃) (R : C₂ ⥤ C₄) (B : C₃ 
 
 namespace CatCommSq
 
+@[reassoc]
```

Comment:
As `NatTrans.naturality` is a simp lemma, this should probably also be a simp lemma:
```suggestion
@[reassoc (attr := simp)]
```